include/boost/wintls/detail/async_handshake.hpp

```diff
@@ -10,16 +10,21 @@
 
 #include <boost/wintls/handshake_type.hpp>
 
+#include <boost/wintls/detail/config.hpp>
 #include <boost/wintls/detail/sspi_handshake.hpp>
 
+#ifdef WINTLS_USE_STANDALONE_ASIO
+#include <asio/coroutine.hpp>
+#else
 #include <boost/asio/coroutine.hpp>
+#endif
```

Comment:
I think it might make more sense to move this to `detail/config.hpp` or perhaps even better, introduce a `detail/coroutine.hpp` header that includes the correct header as well as defines the related macros.